dolly-backend-app/src/main/resources/application.yaml

```diff
@@ -9,6 +9,7 @@ AAD_ISSUER_URI: https://login.microsoftonline.com/62366534-1ec3-4962-8869-9b5535
 CLIENT_ID: f0174d28-cb33-44f9-a7b5-09e52582e601
 HELSEPERSONELL_CLIENT_ID: af686685-d25a-462a-a0ea-e2a348c457d3
 ORGANISASJON_FORVALTER_CLIENT_ID: 80f07134-375b-4df8-be48-2ba877774c6c
+UDI_STUB_CLIENT_ID: 4426ef7d-f148-4d19-8a03-b0d07a451938
```

Comment:
Kanskje du må legge dette som en env config i nais.yml og hente ${UDI_STUB_CLIENT_ID} her da vi vil ha ulike UDI_STUB for dev og prod.

Reply:
Bra sett! Gjorde det sånn i frontend config, men glemte det her ser jeg ja, skal fikse 😄